cmd/helm/testdata/repositories.yaml

```diff
@@ -2,3 +2,8 @@ apiVersion: v1
 repositories:
   - name: charts
     url: "https://charts.helm.sh/stable"
+  - name: example
+    url: "http://example.com"
+  - name: other
+    url: "http://example.com"
```

Comment:
To prevent confusion, would it make sense to name this `otherexample` and `otherexample.com`?

Reply:
Good point.  I've pushed an update which uses `firstexample` and `secondexample`.
I could not use `example` and `otherexample` because the test checks if the name of the repo is part of the output, so we can't have one name (`example`) as part of another name (`otherexample`) or else the test could get confused.